apps/hip/templates/hip/static_page.html

```diff
@@ -7,8 +7,7 @@
 
 {% block sidebar %}
   {# disable left sidebar but preserve the space that it occupies #}
-  <div class="column is-2 pr-0">
-  </div>
+  {% include 'includes/sidebar.html' with hidden_desktop=True %}
```

Comment:
Making this change removes the left sidebar completely (which also makes the comment above incorrect). If you're planning to fix that in the next PR, that is OK. Just want to make sure.

Reply:
I see, working on it.

Reply:
done

Reply:
Restored spacing on screens where the sidebar is hidden on desktop. User is still able to use the sidebar on touch devices.